pkg/tsdb/prometheus/framing_test.go

```diff
@@ -4,71 +4,40 @@ import (
 	"bytes"
```

Comment:
In general  file and package names with `_` is discouraged in golang so I've renamed `json_to_dataframes_test` to `framing_test.go` to illustrate the idea.